BusinessLayer/InventoryCsvManager.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using Microsoft.Extensions.Configuration;
+using Models;
+using Serilog;
+
+namespace BusinessLayer
+{
+    public class InventoryCsvManager : IInventoryManager
+    {
+        private readonly IConfigurationRoot _config;
+        private ILogger _logger;
+
+        public InventoryCsvManager(IConfigurationRoot config, ILogger logger)
+        {
+            if (!File.Exists(config["INVENTORY_PATH"])) throw new FileNotFoundException();
+
+            _config = config;
+            _logger = logger;
+        }
+
+        public Contact AddSerialNumber(Contact contact)
+        {
+            var current = GetLastSerialNumber() + 1;
```

Comment:
current => rename tjis variable to currentSerialNumber

Reply:
or maybe just serialNumber

Reply:
or just do it line 33

Reply:
Deleted variable and added directly SerialNumber in contact.SerialNumber